chemprop/cli/utils/parsing.py

```diff
@@ -119,6 +119,38 @@ def make_datapoints(
     dataset.
     Parameters
     ----------
+    smiss : list[list[str]] | None
```

Comment:
I mostly added the parameter section so that I could make it clear that you pass in `smiss` which is `n x j` and you get out  datapoints that are `j x n`. The fact that we transpose could be confusing. After I added a description of `smiss` and `rxnss` I felt it would be weird to not include the other parameters, though things like `lt_mask` and `gt_mask` are used elsewhere and perhaps it would be better to keep this shorter and not describe their shapes here.